### DISCUSSION TITLE
When i select "deepspeed" and apply model settings in tortoise, it says there was no module found.

### DISCUSSION THREAD

**Discussion by FemBoxbrawl** [rsxdalv/TTS-WebUI#205] [General]
i heard there is also a way to make tortoise tts even faster, so i was wondering if you could look into thise video and somehow add it to the text gen webui? ( i do not know how to code):   https://youtu.be/39RGRfEQFXc?feature=shared

but yeah for some reason deepspeed doesn't download. and lastly, i am asking how i can set my tortoise samples instead of 4 as a minimum, down to 2.

thanks

**Comment by rsxdalv** [comment]
Are you on windows? Because deepspeed requires manual installation on
windows. Jarod has made one but, at least the last few times I checked, has
not made it easily/publicly accessible.

As for the diffusion, 2 might be the same speed as 4, it depends on the
batch size, since these are processed in parallel batches. I don't remember
if I made the fix to set the minimum to be the batch size but as far as i
remember, setting it below 4 would require changing the actual tortoise
code and potentially still be the same speed. (If the batch size is 4 but
you force it to be 2, it just errors).

**Comment by FemBoxbrawl** [comment]
Yes I am on windows. But thanks for letting me know and answering so quick!
i really appreciate it!

I'll most likely figure it out how to install it.

Thanks!
